Rohit/HTML basics remake/form.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>form</title>
+</head>
+<body>
+    <h1>Register</h1>
+ <form autocomplete="on"> 
```

Comment:
Use specific `autocomplete`. Not form. Refer the previous link I gave: [The HTML autocomplete attribute - MDN web docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/autocomplete)